Makefile

```diff
@@ -159,13 +161,12 @@ check-noformat: check-coverage check-py26 check-py27 check-py33 check-py34 check
 
 check: check-format check-noformat
 
-check-fast: lint $(PY35) $(PYPY) $(TOX)
```

Comment:
So actually we do want an explicit Python version here because we're explicitly saying we're calling the py36 tox job, so if the best python moves then that would break.

But there's no reason to also check py35, so I've removed that and made this more explicit.